examples/custom-image-loader/loaders/imgloader.js

```diff
@@ -0,0 +1,14 @@
+module.exports=({ src })=> {
+      
+    // images folder loaders
+    // if(src.indexOf("svg")){
+    //     src="svg/"+src;
+    // }
+    // if(src.indexOf("png")){
+    //     src="image/"+src;
+    // }
+      
+    return `./${src}`; // REPLACE WITH YOUR IMAGE DIRECTORY
+};
```

Comment:
This is not recommended behavior since the `src` is returned as-is without handling `width` or `quality`.

Instead, this should be an external service that can resize and reformat the image.

See https://nextjs.org/docs/api-reference/next/image#loader